src/main/java/summarizer/team/NormalTeamEquipmentSummarizer.java

```diff
@@ -0,0 +1,37 @@
+package summarizer.team;
+
+import equipment.IEquipmentType;
+import team.ITeamMember;
+import equipment.AbstractEquipment;
+import equipment.uniform.UniformEquipmentType;
+import summarizer.SummarizerUtils;
+
+import java.util.List;
+import java.util.Map;
+
+public class NormalTeamEquipmentSummarizer implements ITeamEquipmentSummarizer {
+
+    @Override
+    public TeamSummarizerType getSummaryType() {
+        return TeamSummarizerType.NormalTeamEquipmentSummarizer;
+    }
+
+    @Override
+    public String getTeamSummary(Map<ITeamMember, List<AbstractEquipment>> teamEquipments) {
```

Comment:
the extended summarizer is supposed to output each individual equipment's full description- not a summary of amount and type